src/components/user/list.vue

```diff
@@ -47,7 +47,7 @@ either is an Administrator or has no role. -->
     <user-new v-bind="newUser" @hide="hideModal('newUser')"
       @success="afterCreate"/>
     <user-reset-password v-bind="resetPassword" @hide="hideResetPassword"
-      @success="afterResetPassword"/>
+      @success="afterResetPassword" v-if="!config.oidcEnabled"/> <!-- REVIEW this may not be necessary; could help performance? -->
```

Comment:
Agreed, not necessary, but I think it's a nice change. We could one step further and not even download this modal component, but that's slightly harder (though still not hard, I could show you how if it's of interest).